app/client/components/organizationWrapper.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+import {compose} from 'redux';
+import { updateActiveOrgRequest } from '../reducers/auth/actions'
+import { parseQuery, generateQueryString } from './../utils'
+
+import { browserHistory } from '../app'
+
+const OrganizationWrapper = Component => class extends Component {
+
+    componentDidMount() {
+        //load organisation if param is not matching currently active org
+        let query_params = parseQuery(location.search)
+        if(query_params["org"] && this.props.login.organisationId != query_params["org"]){
+            this.props.updateActiveOrgRequest(query_params["org_name"], query_params["org"])
+        } else {
+            browserHistory.push({
+                search: generateQueryString({org_name: this.props.login.organisationName})
+            })
+        }
+        
+    }
+
+    render() {
+        return <Component {...this.props}></Component>
+    }
```

Comment:
haha I use two spaces... we definitely need to setup a linter and some style rules so there's not constant diffs